### PR TITLE
chore: list new BSC farms

### PR DESCRIPTION
**Pull request by Chef-Yogi**
### Summary
🌾🔄🆕

Add six new farms to `farmsV3` array in `packages/farms/constants/56.ts`. These farms allow users to provide liquidity and earn rewards for various token pairs on the PancakeSwap platform.

> _`farmsV3` expands_
> _Six new pools for swapping tokens_
> _Autumn harvest time_

### Walkthrough
*  Add six new farms to the `farmsV3` array with the latest pool data ([link](https://github.com/pancakeswap/pancake-frontend/pull/7399/files?diff=unified&w=0#diff-b68c662a4cbcb945733d7c43d7c670219954b0f8395016547312ad2d6ad6a066R46-R87))

### CODE REVIEW

File: packages/farms/constants/56.ts [hunk 1]
```diff
@@ -42,6 +42,48 @@ export const farmsV3 = defineFarmV3Configs([
...
+  {
+    pid: 59,
+    token0: bscTokens.tusd,
+    token1: bscTokens.busd,
+    lpAddress: '0xdf0c1C30e8C1aE3f189f6E6ef248d71977F7BE29',
+    feeAmount: FeeAmount.LOWEST,
+  },
+  {
+    pid: 58,
+    token0: bscTokens.bnb,
```
Comment (ChefMomota) [inline packages/farms/constants/56.ts:54]: bnb or wbnb ?

File: packages/farms/constants/56.ts [hunk 2]
```diff
@@ -42,6 +42,48 @@ export const farmsV3 = defineFarmV3Configs([
...
+    pid: 56,
+    token0: bscTokens.usdt,
+    token1: bscTokens.btcb,
+    lpAddress: '0x6ee3eE9C3395BbD136B6076A70Cb6cFF241c0E24',
+    feeAmount: FeeAmount.MEDIUM,
+  },
+  {
+    pid: 55,
+    token0: bscTokens.btcb,
+    token1: bscTokens.bnb,
```
Comment (ChefMomota) [inline packages/farms/constants/56.ts:76]: same as above

File: packages/farms/constants/56.ts [hunk 3]
```diff
@@ -42,6 +42,48 @@ export const farmsV3 = defineFarmV3Configs([
...
+    lpAddress: '0x6bbc40579ad1BBD243895cA0ACB086BB6300d636',
+    feeAmount: FeeAmount.LOW,
+  },
+  {
+    pid: 54,
+    token0: bscTokens.eth,
+    token1: bscTokens.bnb,
```
Comment (ChefMomota) [inline packages/farms/constants/56.ts:83]: same as above

Comment (Chef-Yogi) [reply packages/farms/constants/56.ts:83]: the bnb here is wbnb actually
CleanShot 2023-07-24 at 12 50 26@2x

**Comment by Chef-Yogi** [comment]
@ChefMomota  Done

**Review by ChefMomota — APPROVED** [review]

**Pull request merged**